AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift

```diff
@@ -1,39 +1,38 @@
+
 //
 //  ********************************************************
 //  *************   DO NOT COMMMIT THIS FILE   *************
 //  *************         DO NOT SHARE IT      *************
 //  ************* It Has Sensitive Credentials *************
 //  ********************************************************
 //
-//  Dummy credentials file.
-//
 //  AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift
 //
 //  AmahiAnywhere
 //
 //  Copyright © 2018 Amahi. All rights reserved.
 //
+
 import Foundation
 
 struct ApiConfig {
     
-    static let baseUrl =                       "https://example.com"
-    static let proxyUrl =                       "https://example.com"
-    static let appID =                         "AAAAAAAA"
+    static let baseUrl =                       "xyz"
+    static let proxyUrl =                      "xyz"
+    static let appID =                         "xyz"
+    
+    private static let CLIENT_ID =              "xyz"
+    private static let CLIENT_SECRET =          "xyz"
     
-    private static let clientId =              "C-I-D"
-    private static let clientSecret =          "C-S-T"
     
     static func oauthCredentials(username: String, password: String) -> [String : String] {
         
-        let parameters =                          ["client_id": clientId,
-                                                   "client_secret": clientSecret,
-                                                   "username" : username,
-                                                   "password" : password ]
-        
-        fatalError("You may need to get dev credentials to be able to properly login into amahi.org." +
```

Comment:
I rather if the fatalError is left intact, so that people know right away.

Reply:
So I have to remove all the code and only leave `FatalError` in this file right?

Reply:
I did not say that. I specifically talked about the one line of the fatalError.